spec/cookbook_spec.rb

```diff
@@ -19,36 +19,51 @@
 require 'between_meals/changes/cookbook'
 require 'between_meals/changeset'
 require 'logger'
+require 'find'
 
 describe BetweenMeals::Changes::Cookbook do
   let(:logger) do
     Logger.new('/dev/null')
   end
   let(:cookbook_dirs) do
-    ['cookbooks/one', 'cookbooks/two']
+    ['cookbooks/one', 'cookbooks/two', 'cookbooks/three']
+  end
+  let(:repo_path) do
+    "#{ENV['HOME']}/devfiles"
+  end
+
+  class Repo
+    def repo_path
+      "#{ENV['HOME']}/devfiles"
+    end
   end
 
   fixtures = [
     {
       :name => 'empty filelists',
       :files => [],
       :result => [],
+      :result_with_symlink_tracking => [],
     },
     {
       :name => 'modifying of a cookbook',
       :files => [
         {
           :status => :modified,
-          :path => 'cookbooks/two/cb_one/recipes/test.rb',
+          :path => 'cookbooks/one/cb_one/recipes/test.rb',
         },
         {
           :status => :modified,
-          :path => 'cookbooks/two/cb_one/metadata.rb',
+          :path => 'cookbooks/one/cb_one/metadata.rb',
         },
       ],
       :result => [
         ['cb_one', :modified],
       ],
+      :result_with_symlink_tracking => [
+        ['cb_one', :modified],
```

Comment:
wut? we see it twice?!

Reply:
ya we see the original, and the symlinked location. This is unlikely a real use case, as no one with have a symlink to a cookbook that is in their path.

But you might have update to a cookbook that is not in one of your cookbook_dirs and then would only see the one.